nnpackage/schema/circle_schema.fbs

```diff
@@ -1003,6 +1005,15 @@ table InstanceNormOptions {
   fused_activation_function:ActivationFunctionType;
 }
 
+table MaxPoolWithArgMaxOptions {
+  padding:Padding;
+  stride_w:int;
+  stride_h:int;
+  filter_width:int;
+  filter_height:int;
```

Comment:
Maybe add fused activation, like in ordinary pooling?
Also, maybe name output_type. some thing like "indexing_type"?